platforms/platform_db/platform_defaults/defaults_local_memory.json

```diff
@@ -17,18 +17,32 @@
       [
          "Like the same field in CCI-P, this is the recommended number",
          "of times an AFU should register local memory signals before use",
-         "in order to make successful timing closure likely.  For Avalon",
-         "MM interfaces this value is typically zero since registering",
-         "Avalon MM's waitrequest protocol is complex."
+         "in order to make successful timing closure likely.",
+         "",
+         "When a local memory clock crossing is also requested, using",
```

Comment:
why will a local memory clock crossing be requested? different clock domain between FIU and AFU?